Simplenote/Classes/SPNoteTableViewCell.swift

```diff
@@ -9,15 +9,15 @@ class SPNoteTableViewCell: UITableViewCell {
 
     /// Master View
     ///
-    @IBOutlet private var containerView: UIView!
+    @IBOutlet private var titleTextView: UITextView!
 
     /// Accessory StackView
     ///
-    @IBOutlet private var accessoryStackView: UIStackView!
+    @IBOutlet private var bodyTextView: UITextView!
```

Comment:
I'm curious about the choice of UITextView vs UILabel. Seems like these are just for display.  UILabel is lighter weight, supports attributed strings, doesn't have its own scroll view hierarchy embedded, and is mostly (completely?) configurable from the xib.

Reply:
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2019-09-25 at 16 49 25](https://user-images.githubusercontent.com/1435271/65642314-947f7880-dfb4-11e9-846f-e8753a26bc7e.png)
I think UILabel's might be more #00ff00 as well.